Source/Parser/wp_parser_y.h

```diff
@@ -191,9 +191,9 @@ wp_call_f1 (enum wp_f1_t type, T a)
     case WP_COSH:        return std::cosh(a);
     case WP_TANH:        return std::tanh(a);
     case WP_ABS:         return amrex::Math::abs(a);
-    case WP_POW_M3:      return 1.0/(a*a*a);
-    case WP_POW_M2:      return 1.0/(a*a);
-    case WP_POW_M1:      return 1.0/a;
+    case WP_POW_M3:      return static_cast<amrex::Real>(1.0)/(a*a*a);
+    case WP_POW_M2:      return static_cast<amrex::Real>(1.0)/(a*a);
+    case WP_POW_M1:      return static_cast<amrex::Real>(1.0)/a;
```

Comment:
Simpler for literals:
```suggestion
    case WP_POW_M3:      return amrex::Real(1.0)/(a*a*a);
    case WP_POW_M2:      return amrex::Real(1.0)/(a*a);
    case WP_POW_M1:      return amrex::Real(1.0)/a;
```